nmpolicy/types/json.go

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
If you try to Unmarshal current `types` package it fails since `desiredState` cannot be treatd as an string but as a yaml, same as we do at kubernetes-nmstate, this was going to be art of the `cli` PR before starting the docs but looks like docs has more priority so we need to put it here (it's needed to run the examples).